SPECS/moby-engine/moby-engine.spec

```diff
@@ -116,13 +117,18 @@ fi
 %files
 %license LICENSE NOTICE
 %{_bindir}/dockerd
+# docker-proxy symlink in bindir to fix back-compat
+%{_bindir}/docker-proxy
 %{_libexecdir}/docker-proxy
 %dir %{_sysconfdir}/docker
 %config(noreplace) %{_sysconfdir}/docker/daemon.json
 %{_sysconfdir}/*
 %{_unitdir}/*
 
 %changelog
+* Wed May 01 2024 Henry Beberman <henry.beberman@microsoft.com> - 24.0.9-3
```

Comment:
Please put the fast-track change in front of the main-only one. Since fast-track changes are already published, it's best we don't re-order the history in future releases. As long as they are not published, the order of the main-only changes doesn't really matter.